Dockerfile

```
@@ -1,6 +1,9 @@
 FROM  node:10-alpine
 MAINTAINER OhMyForm <admin@ohmyform.com>
 
+# Create a group and a user with name "ohmyformUser".
+RUN addgroup -g 9999 ohmyformGroup && adduser -u 99999 -D -g ohmyformGroup ohmyformUser
```

```suggestion
RUN addgroup -g 9999 ohmyform && adduser -u 99999 -D -G ohmyform ohmyform
```
perhaps this may fulfil @wodka suggestion, unable to test atm due to work